### DISCUSSION TITLE
How To Find The numbers of elements in one object Including nested objects.

### DISCUSSION THREAD

**Discussion by beemibrahim** [nlohmann/json#3269] [Q&A] [answered]

**Comment by nlohmann** [comment, marked as answer]
There is no member function for this. You need to recurse through the value yourself. There is an example given in https://stackoverflow.com/a/46630394/266378 that can be adjusted for your counting usecase.

**Reply by nlohmann** [reply]
@beemibrahim Do you need further assistance with this issue?

**Reply by beemibrahim** [reply]
no thanks
